data/json/items/comestibles/carnivore.json

```diff
@@ -109,6 +109,31 @@
     "delete": { "flags": [ "SMOKABLE" ] },
     "vitamins": [ [ "vitA", 0 ], [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ], [ "vitB", 0 ] ]
   },
+  {
+    "id": "mutant_meat",
+    "type": "COMESTIBLE",
+    "copy-from": "meat",
+    "name": "chunk of mutant meat",
+    "name_plural": "chunks of mutant meat",
+    "description": "Meat from a heavily mutated animal.  It has an unsettling loose and spongy texture, but smells… mostly normal.  Seems safe to eat, if you cook it.",
```

Comment:
Suggested alterations for better lore information:
"Meat from a heavily mutated animal.  It has an unsettling loose and spongy texture, but smells… mostly normal.  There are strange tangles and formations in it that don't appear natural at all: bits of bone and hair crusted up inside the muscle, as if trying to form another organism. Still, seems safe to eat, if you cook it and remove the worst parts.",

Reply:
Ooh, flavorful! I like it.

Random thought: It would be really cool if we could gate more detailed descriptions like this behind high intelligence/skill.

Reply:
Yes it would